civiform_config.example.sh

```diff
@@ -76,6 +77,18 @@ export CIVIFORM_APPLICANT_AUTH_PROTOCOL=""
 # Deployment-specific Civiform configuration
 #################################################
 
+# REQUIRED
+# The short name for the civic entity. Ex. "Seattle"
+export CIVIC_ENTITY_SHORT_NAME=""
```

Comment:
question (blocking): Didn't we just remove these because they are admin-writeable? https://github.com/civiform/civiform-deploy/pull/46/files

Reply:
We did! However the deploy system wouldn't let Tom proceed without them. Maybe that's not still the case?

Reply:
reverted

Reply:
ah it's erroring out because it's still "required" in `variable_definitions.json`: https://github.com/civiform/cloud-deploy-infra/blob/dc2470a8297b6525125fb38dea6c87ba30152c31/cloud/shared/variable_definitions.json#L8

we can either remove "required" from that now, or just wait a few weeks and remove those when we do the other cleanup from auto-generation.

Reply:
Let's make them optional: https://github.com/civiform/cloud-deploy-infra/pull/221